ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-stable.yaml

```diff
@@ -516,6 +516,17 @@ tests:
     test:
     - ref: openshift-extended-test
     workflow: cucushift-installer-rehearse-gcp-ipi-private
+- as: e2e-rosa-nightly-sts
+  interval: 500h
```

Comment:
@yasun1 suggest not to use interval and change to use cron to control the test exactly.

Reply:
make sense. The job is planned to run every 3 weeks(per sprint). I will update it to `0 9 * * */3`.

Reply:
The above cron is not as expected, it means `At 09:00 on every 3rd day-of-week from Wednesday through Sunday`. There is no good way to achieve `run on Wednesday every 3 weeks(per sprint)`, just define it to `0 8 * * 3`, which is `At 08:00 on Wednesday`. Later, let me discuss with the manager about this.